src/utils/source-search.js

```diff
@@ -58,8 +64,12 @@ function ignoreWhiteSpace(str) {
  * @memberof utils/source-search
  * @static
  */
-function searchOverlay(query) {
-  query = new RegExp(escapeRegExp(ignoreWhiteSpace(query)));
+function searchOverlay(query, regexMatch) {
+  if (regexMatch && query !== "") {
+    query = new RegExp(query);
```

Comment:
@bomsy Hmm yeah that seemed to do the trick, highlight is actually showing up. This is a complicated feature 😆  Thanks!